openmetadata-service/src/main/java/org/openmetadata/service/secrets/converter/WorkflowClassConverter.java

```diff
@@ -30,7 +31,8 @@ public WorkflowClassConverter() {
   public Object convert(Object object) {
     Workflow workflow = (Workflow) JsonUtils.convertValue(object, this.clazz);
 
-    tryToConvertOrFail(workflow.getRequest(), List.of(TestServiceConnectionRequest.class))
+    tryToConvertOrFail(
+            workflow.getRequest(), List.of(TestServiceConnectionRequest.class, Map.class))
```

Comment:
allow workflow configurations to be arbitrary maps so they can be extended on collate without exposing the interface on OpenMetadata